src/main/java/edu/harvard/iq/dataverse/importer/metadata/mxrdr/MetadataField.java

```diff
@@ -0,0 +1,44 @@
+package edu.harvard.iq.dataverse.importer.metadata.mxrdr;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.UnaryOperator;
+
+/**
+ * Class designed to hold metdata fields with filter made to extract desired field.
+ */
+public class MetadataField {
+
+    private final String name;
+    private UnaryOperator<String> fieldFilter;
+    private final List<MetadataField> childFields = new ArrayList<>();
+
+    // -------------------- CONSTRUCTORS --------------------
+
+    public MetadataField(String name, UnaryOperator<String> fieldFilter) {
+        this.name = name;
+        this.fieldFilter = fieldFilter;
+    }
+
+    public MetadataField(String name) {
+        this.name = name;
+    }
+
+    // -------------------- GETTERS --------------------
+
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Filter function which helps extract metadata field from String line.
+     * @return extracted field value
+     */
+    public UnaryOperator<String> getFieldFilter() {
```

Comment:
`which helps` doesn't tell much. How it helps? Maybe some example? What it should receive and what will be the result? Maybe we could mention what it means if function will return empty string?